airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/models/__init__.py

```diff
@@ -1,27 +1,2 @@
-"""
```

Comment:
same here

Reply:
I excluded `__init__.py` files because a lot of them were blank and I wasn't sure how to write a regex that made spotless realize that. I could add a bogus comment or import I guess, but since they were blank it didn't seem important?